datasets/inception_subset.py

```diff
@@ -0,0 +1,174 @@
+########################################################################################################
+#                                                                                                      #
+#                           Inception Subset                                                           #
+#                                                                                                      #
+########################################################################################################
+
+# TORCH
+import torch
+from torch.utils.data import Dataset, TensorDataset
+
+# Python Packages
+import numpy as np
+# import glob
+from datetime import datetime
+from typing import List, Type, Tuple, Iterator
+from collections import defaultdict  # Nested dictionaries in python
+
+# SKLearn
+from sklearn.model_selection import train_test_split
+from sklearn.model_selection import StratifiedKFold
+
+# PNN SPECIFIC
+from datasets.labels import labels
+from train import parameters
+
+
+class Modalities(Dataset):
+    """
+    A dataset class that lets the user decides which modalities to use.
+    """
+
+    def __init__(self, root_dir: str,
+                 exp_name: str,
+                 start_date,
+                 end_date,
+                 **plant_files_dict):
+        """
+        :param root_dir: path to the experiment directory
+        :param exp_name: the experiment we want to use
+        :param mods: modalities to be in the dataset, as dictionaries of initialization arguments
+
+        """
+        self.experiment = exp_name
```

Comment:
- [ ] rename to `self.experiment_name`
```suggestion
        self.experiment_name: str = exp_name
```